images/fcos-bastion-image/root/usr/lib/systemd/system/squid-proxy.service

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=Creates the squid proxy container
+Wants=network-online.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/usr/share/bastion_services/squid/squid_systemd.conf
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStartPre=/usr/bin/podman login quay.io
+ExecStartPre=-/bin/sh -c '/usr/bin/mkdir -p ${PODMAN_SQUID_BASEDIR}/{cache,log,lib}'
+ExecStartPre=-/bin/sh -c '/usr/bin/podman run --rm --interactive -v "${PODMAN_SQUID_BASEDIR}/lib:/var/lib:Z" ${PODMAN_SQUID_IMAGE} bash < /usr/share/bastion_services/squid/create_ssl_db.sh'
```

Comment:
the whole /var/lib as a volume is a lot. Can we reduce to the required dirs?

The current container runs with the following bind mounts:
```
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/srv/squid/etc",
                "Destination": "/etc/squid",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/srv/squid/cache",
                "Destination": "/var/spool/squid",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/srv/squid/log",
                "Destination": "/var/log/squid",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            }
        ],
```